### ISSUE TITLE
Resolutions higher than 720p

### ISSUE THREAD

**Issue by FireMasterK** [pluja/Parasitter#48]
Currently, the maximum resolution seems to be 720p?

**Comment by pluja** [comment]
Yes. This needs some work! I think this shouldn't be hard to address

**Comment by fromxt** [comment]
Can we  automatically select the available dash quality or something like invidious?

**Comment by pluja** [comment]
Not right now, but I will be working on it.

**Comment by FireMasterK** [comment]
We should ideally have a quality selector like how newpipe does, including support for webm.

**Comment by pluja** [comment]
Yes, I think I know how we can do this. I will work on it on the following days.

**Comment by FireMasterK** [comment]
I did a lot of research on this recently, you'll need to generate a dash manifest for this to even work, otherwise you'll get a video without any audio.

**Comment by FireMasterK** [comment]
And it will be impossible for clients without javascript to play these streams.

**Comment by pluja** [comment]
Hm... I see. Thank you! I will leave this issue for now, I am focusing on having everything working fine for now as I have a limited time!

**Comment by unixfox** [comment]
I'm really interested into this issue. Once this will be implemented I might consider switching https://yewtu.be from Invidious to Yotter.

**Comment by pluja** [comment]
This needs some work, and I currently don't have any time to code... Maybe in a week or so I will be able to start dedicating time to Yotter and get all these features done.

**Comment by FireMasterK** [comment]
This is fortunately possible to implement with `youtube-dlc`. However, from my [research](https://github.com/videojs/mux.js/search?q=vp9), `videojs` does not support remuxing for WEBM streams.

This is also the reason why invidious only has support for mp4 files. (and it's 50kbps audio bitrate)

You can take a look at https://github.com/GilgusMaximus/yt-dash-manifest-generator to see how it should be done to mp4 files.

This is why I've decided to try to create yet another proxy. One that will mux 2 streams using `ffmpeg`.

**Comment by FireMasterK** [comment]
However, I would like some feedback for the hacky fix I made in https://piped.kavin.rocks/, another WIP alternative youtube frontend.

Just tell me what browser you use, if the audio is stuttering during videoplayback, and a log of the numbers you see in your dev tools console.

This has support for WebM streams and uses a similar approach to what cloudtube does.

**Comment by fadster** [comment]
@FireMasterK I'm not getting any audio on Piped.
